The Globe and Mail.js

```diff
@@ -1,223 +1,189 @@
-			for (var i=0; i<authors.length; i++) {
-				var author = authors[i];
-				var words = author.split(" ");
-				
-				for (var j=0; j<words.length; j++) {
-					if (words[j] != "") {
-						words[j] = words[j][0].toUpperCase() + words[j].substr(1).toLowerCase();
-					}
-				}
-				author = words.join(" ");
-				newItem.creators.push(Zotero.Utilities.cleanAuthor(author, "author"));			
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
 			}
-		}
-	}	
-	
-	newItem.url = doc.location.href;
-	newItem.complete();
-	
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
 }
 
-function doWeb(doc, url) {
-	var namespace = doc.documentElement.namespaceURI;
-	var nsResolver = namespace ? function(prefix) {
-		if (prefix == 'x') return namespace; else return null;
-	} : null;
+function scrape(doc, url) {
 	
-	var articles = new Array();
+	newItem = new Zotero.Item("newspaperArticle");
+	newItem.url = url;
+
+	//get headline
+	var title = ZU.xpathText(doc, '//article/h1');
+	if (!title) title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	newItem.title = title;
+
+	//get abstract
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@property="og:description"]/@content');
 	
-	if (detectWeb(doc, url) == "multiple") {
-		var items = new Object();
-		var next_title = new Array();
-		
-		var titlesCount = doc.evaluate('count (//h3[@class="storyLink"]/a)', doc, nsResolver, XPathResult.ANY_TYPE, null);
-		var titles = doc.evaluate('//h3[@class="storyLink"]/a', doc, nsResolver, XPathResult.ANY_TYPE, null);		
-		
-		for (i=0; i < titlesCount.numberValue; i++) {
-			next_title = titles.iterateNext();
-			
-			if (next_title.href.match("story")) {
-				items[next_title.href] = next_title.textContent;
+	//get date
+	var xpathdate = '//time[@itemprop="datePublished"]';
+	var date = ZU.xpathText(doc, xpathdate);
+	if (date) {
+		newItem.date = ZU.strToISO(date);
+	}
+
+	//get author
+	var authors = ZU.xpath(doc, '//meta[@itemprop="author"]/@content');
+	for (var i in authors){
+		newItem.creators.push(ZU.cleanAuthor(authors[i].textContent, "author"));
+	}
+
+	var publishers = ZU.xpath(doc, '//article/header//div[@itemprop="publisher"]//p');
+	for (var i in publishers ){
+		newItem.creators.push( {
+		"lastName" : publishers[i].innerText,
+		"creatorType" : "contributor",
+		"fieldmode" : 1
+		});
+	}
+
+	newItem.language = ZU.xpathText(doc, '//meta[@http-equiv="Content-Language"]/@content');
+	
+	newItem.section = ZU.xpathText(doc, '//meta[@name="article:type"]/@content');
+
+	var xpathtags = ZU.xpathText(doc, '//meta[@name="keywords"]/@content');
+	newItem.tags = xpathtags.split(";").filter(function(tag) {return tag.length != 0});
+
+	var related_links = ZU.xpathText(doc, '//article//li/p/a/@href');
+	if(related_links)
+		newItem.notes = related_links.split(",").filter(function(tag) {return tag.length != 0});
+
+	newItem.attachments = ({
+		url: url,
```

Comment:
Documentation is here: https://www.zotero.org/support/dev/translators/coding#attachments (the last bit). 
But the test indeed doesn't look right. I have never seen that behavior in Scaffold, would have to check. What's the exact set-up you're running that on?